### PR TITLE
docs: bash in user job images

### PR DESCRIPTION
**Pull request by tiborsimko**

### CODE REVIEW

File: docs/advanced-usage/containers/docker/index.md [hunk 1]
```diff
@@ -40,15 +40,42 @@ WORKDIR /code
...
+encoded/decoded job commands and parameters between REANA workflow
+orchestration components, the job execution components and the compute backend
+itself, so that the job execution behaviour would be consistent across
+Kubernetes, HTCondor, Slurm backends for both Docker and Singularity execution
+wrappers.
+
+Therefore, please make sure that your Docker images contain the `bash` shell
+executable, even if it may not be the default shell.
+
+For example, if you would ilke to use the tiny Alpine image, which uses `ash`
```
Comment (mdonadoni) [inline docs/advanced-usage/containers/docker/index.md:67]:
```suggestion
For example, if you would like to use the tiny Alpine image, which uses `ash`
```

File: docs/advanced-usage/containers/docker/index.md [hunk 2]
```diff
@@ -40,15 +40,42 @@ WORKDIR /code
...
+shell by default, you can add `bash` in your `Dockerfile` as follows:
+
+```{ .Dockerfile .copy-to-clipboard }
+FROM alpine:3.17
+RUN apk add bash
+```
+
+The `bash` shell is relatively wide spread, so your base images might already
```
Comment (mdonadoni) [inline docs/advanced-usage/containers/docker/index.md:75]:
```suggestion
The `bash` shell is relatively widespread, so your base images might already
```

File: docs/advanced-usage/containers/docker/index.md [hunk 3]
```diff
@@ -40,15 +40,42 @@ WORKDIR /code
...
+orchestration components, the job execution components and the compute backend
+itself, so that the job execution behaviour would be consistent across
+Kubernetes, HTCondor, Slurm backends for both Docker and Singularity execution
+wrappers.
+
+Therefore, please make sure that your Docker images contain the `bash` shell
+executable, even if it may not be the default shell.
+
+For example, if you would ilke to use the tiny Alpine image, which uses `ash`
+shell by default, you can add `bash` in your `Dockerfile` as follows:
```
Comment (mdonadoni) [inline docs/advanced-usage/containers/docker/index.md:68]:
```suggestion
shell by default, you can add a command in your `Dockerfile` to install `bash` as follows:
```
WDYT?

File: docs/advanced-usage/containers/docker/index.md [hunk 4]
```diff
@@ -40,15 +40,42 @@ WORKDIR /code
...
+
+```{ .Dockerfile .copy-to-clipboard }
+FROM alpine:3.17
+RUN apk add bash
+```
+
+The `bash` shell is relatively wide spread, so your base images might already
+contain it. If the requirement on the `bash` presence causes troubles, and you
+cannot add it to your images, then please get in touch.
```
Comment (mdonadoni) [inline docs/advanced-usage/containers/docker/index.md:77]: What about _"Please get in touch with us if requiring `bash` to be present in your Docker images causes you any trouble."_ ?

Comment (tiborsimko) [reply docs/advanced-usage/containers/docker/index.md]: I have reformulated the whole final paragraph to make it more easily readable as a "conclusion thoughts".

**Review by mdonadoni — APPROVED** [review]
Looks good!

**Pull request merged**